public/test/matchers/toEmitValues.test.ts

```diff
@@ -0,0 +1,125 @@
+import { interval, Observable, of, throwError } from 'rxjs';
+import { map, mergeMap, take } from 'rxjs/operators';
+import { OBSERVABLE_TEST_TIMEOUT_IN_MS } from './types';
+
+describe('toEmitValues matcher', () => {
+  describe('failing test (need to skip these for obvious reasons)', () => {
+    describe.skip('passing null in expect', () => {
```

Comment:
why the `skip` are here, is this by accident?

Reply:
How do I test a failing test without failing the tests?

Reply:
I would write it like this
```
        try {
          await expect(observable).toEmitValues([1, 2, 3]);
        } catch (error) {
          expect(error.matcherResult.message()).toBe('');
        }
```

Reply:
> I would write it like this
> 
> ```
>         try {
>           await expect(observable).toEmitValues([1, 2, 3]);
>         } catch (error) {
>           expect(error.matcherResult.message()).toBe('');
>         }
> ```

Have you tried that @zoltanbedi? I thought you couldn't catch failing excepts?

Reply:
> Love this PR! Awesome job Hugo. Left some comments to consider.

Thank you Zoltan! If you love it a lot maybe you could approve it? ;)

Reply:
Yes, I tried it

Reply:
This is glorious news to me, thank you Zoltan! I'll look into all your comments after the Gardening week.

Reply:
@zoltanbedi I struggled quite a bit with this, as your suggestion wouldn't work if the test was wrong and passed for some reason. Then it would never reach the catch statement, I know a bit overkill perhaps but I want these tests to be solid. Let me know what you think of the update.

Reply:
I like your approach there.